api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -72,7 +72,7 @@ public void setImage(String image) {
 
     @Description("List of topics which are included for mirroring. This option allows any regular expression using Java-style regular expressions." +
             "Mirroring two topics named A and B can be achieved by using `--whitelist 'A|B'`. Or you could mirror all topics using `--whitelist '*'`.")
```

Comment:
I know it's not strictly part of this, but can we change the doc "Or you could mirror all topics using `--whitelist '*'`" → "Or, as a special case, you can mirror all topics using the whitelist `*`"? That makes clear that `*` isn't magically a regex, and also removes the reference to `--whitelist` which I don't think is helpful here.